glib/sys/Gir.toml

```
@@ -71,12 +71,49 @@ status = "generate"
     name = "unix_get_passwd_entry"
     cfg_condition = "unix"
     [[object.function]]
-    name = "win32_get_package_installation_directory_of_module"
-    manual = true
-    cfg_condition = "windows"
-    [[object.function]]
     name = "get_user_state_dir"
     version = "2.72"
+    [[object]]
+    name = "GLib.Win32.*"
```

```suggestion
    name = "GLib.*"
```

and then fold that into the one above. There is no Win32 object